src/Containers/Box.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface BoxProps {
+    text?: string;
+    align?: string;
+    selected?: boolean;
+    children?: React.ReactNode;
+    padding?: string;
+    margin?: string;
+    bold?: boolean;
+    size: string;
+}
+
+export const Box: React.FC<BoxProps> = ({
+    text,
+    children,
+    size = 'default',
+    ...props
+}): React.ReactElement => {
+    return (
+        <BoxDiv {...props} size={size}>
+            {text}
+            {children}
+        </BoxDiv>
+    );
+};
+
+const BoxDiv = styled.div<BoxProps>`
+    ${({ theme, ...props }): string => `
+        box-shadow: ${theme.depth[2]};
+        border-radius: ${theme.dimensions.radius};
+        background-color: ${props.selected && theme.colors.primary};
```

Comment:
tbh when would it be selected? what does it mean for a generic box to be selected? isnt it better just to have a color prop?

Reply:
and if it does make sense, then i guess box is a really bad name for this